content/blog/entries/design-alternatives/contents.lr

```diff
@@ -0,0 +1,27 @@
+title: Design ALT + ernatives
+---
+categories:
+outreachy
+design
+open-source
+---
+author: dhruvi16
+---
+series: ux-design-cycle
+---
+pub_date: 2020-06-22
+---
+body:
+
+
+In this article, I will be talking about the next step of the UX design cycle and that is **Design Alternative**.
+
+This article is inspired by a Coursera course, [Introduction to User Experience](https://www.coursera.org/learn/user-experience-design/).
+
+At this point in the design cycle, we have enough knowledge of the problem space as we have completed the first step — Resource Gathering. By problem space, I mean we have enough knowledge about the user, the task, the current way of accomplishment, and other relevant data. Now, the goal of the next step is to use the data and come up with a better design alternative which is both useful and usable. By useful, I mean an interface that accomplishes the task, and by usable, I mean an interface that is efficient, effective, and satisfying.
+
+The goal of processing the data to achieve a better interface or system does not have an isolated environment. The system affects an individual, a group of people, and even a society. A designer needs to keep in mind how his/her designs benefit them all.
```

Comment:
Personally I prefer 'their' instead of 'his/her'.
```suggestion
The goal of processing the data to achieve a better interface or system does not have an isolated environment. The system affects an individual, a group of people, and even a society. A designer needs to keep in mind how their designs benefit them all.
```